front-end-poc/src/Shared/api/audit.api.ts

```diff
@@ -1,9 +1,10 @@
-import {callOnce, httpClient} from "./httpClient";
-import {AxiosRequestConfig} from "axios";
+import { callOnce, httpClient } from "./httpClient";
+import { AxiosRequestConfig } from "axios";
```

Comment:
I really hope you did not fix all those spacing manually :D
We surely need a Prettier configuration with shared formatting rules. Will add it asap.

Reply:
Nope, VSCode's default prettifier was responsible.